index.js

```diff
@@ -18,22 +17,22 @@ module.exports = class ngAnnotateCompiler {
   compile(file) {
     if (this.isIgnored(file.path)) return Promise.resolve(file)
     const options = withInFile(this.config, file.path)
-    const {src, map = file.map, errors} = ngAnnotate(file.data, options)
-    if (errors) return Promise.reject(errors)
-    return Promise.resolve(assignToCopy(file, {data: src, map}))
+    const results = ngAnnotate(file.data, options)
```

Comment:
Nitpick: maybe the name could be a little more descriptive? What about `annotated`?